modules/swagger-models/src/main/java/com/wordnik/swagger/models/properties/RefProperty.java

```diff
@@ -56,7 +57,7 @@ public String getSimpleRef() {
   }
 
   public static boolean isType(String type, String format) {
```

Comment:
This was always `false` as `"$ref" != "ref"`.
Need this change for `PropertyBuilder.Processor.REFERENCE.isType`